js/src/tooltip.js

```diff
@@ -273,7 +272,7 @@ class Tooltip extends BaseComponent {
 
     tip.classList.add(CLASS_NAME_SHOW)
 
-    const customClass = typeof this._config.customClass === 'function' ? this._config.customClass() : this._config.customClass
+    const customClass = this._parseContent(this._config.customClass)
```

Comment:
Now the context is changed when calling `this._config.customClass` :thinking:

Reply:
supposing it will have more info like this, in opposite way, why to use a function? 
Do you find it wrong?

Reply:
No, I was just mentioning that `this` will be changed now.